route-rs-runtime/src/element/dec_ip_ttl.rs

```diff
@@ -0,0 +1,52 @@
+use crate::element::Element;
+use smoltcp::wire::*;
+
+/*
+  DecIPTTL
+  Decrements the TTL of an IPv4 packet
+*/
+#[derive(Default)]
+pub struct DecIPTTLElement {}
```

Comment:
Rust conventions are to avoid having a lot of capital letters next to each other in an identifier. For instance, `DecIpv4HopLimit` is probably a better name. (This is to avoid inconsistency like Java's `URLConnection` and `HttpURLConnection`.)

Reply:
Is DecIpv4Hop sufficient? DecIpv4HopLimitElement just feels a bit wordy

Reply:
How about no `Element` suffix? It's in the `element` module, we don't need to redundantly call it an `Element`.

Reply:
Makes sense, updated